apps/myaccount/src/components/profile/profile.tsx

```diff
@@ -690,10 +769,7 @@ export const Profile: FunctionComponent<ProfileProps> = (props: ProfileProps): R
             }
 
             return (
-                isFeatureEnabled(
-                    featureConfig?.personalInfo,
-                    AppConstants.FEATURE_DICTIONARY.get("PROFILEINFO_MOBILE_VERIFICATION")
-                ) && checkSchemaType(schema.name, "mobile")
+                isMobileVerificationEnabled && checkSchemaType(schema.name, "mobile")
```

Comment:
isFeatureEnabled util is for conditionally show/hide a feature from a deployment level. Is `isMobileVerificationEnabled` for the same purpose?

if `isMobileVerificationEnabled` to show/hide the UI in tenant level, let's keep the isFeatureEnabled util as well for deployment level configuration.

Reply:
yes `isMobileVerificationEnabled` will be used to show/hide the UI from tenant level. Will keep this deployment level config as wel